### PR TITLE
Add rotation function 

### PR DESCRIPTION
**Pull request by KjetilIN**
# What? 

PR for a rotation function for pitch and roll. Also able to change.

### CODE REVIEW

File: Source/UserLocServer/Private/MovableCamera.cpp
Comment (KjetilIN) [inline Source/UserLocServer/Private/MovableCamera.cpp:143]: dRoll can be a configurable variable @haugeSander 

default value should be 0.09

File: Source/UserLocServer/Private/MovableCamera.cpp
Comment (KjetilIN) [inline Source/UserLocServer/Private/MovableCamera.cpp:158]: dPitch can be a configurable variable @haugeSander 

default value should be 0.06

**Review by KjetilIN — COMMENTED** [review]
Note that the presets for rotation needs refactor before merging this in

**Comment by KjetilIN** [comment]
Do not merge. Will change the function.

**Comment by KjetilIN** [comment]
The new equation that should be implemented and tested with. 

```math
\begin{equation}
    \begin{aligned}
        \text{Roll}(X, Z) = \frac{\Delta R}{Z} \left(X - \frac{W_{UE}}{2}\right)^2 + C_R \\
        \text{Pitch}(Y, Z) = \frac{\Delta P}{Z} \left(X - \frac{H_{UE}}{2}\right)^2 + C_P
    \end{aligned}
\end{equation}
```

Potential problem is that the Z value increases to much. Need to test how this will work but there is three ideas to fix this: 
- take the logarithmic scale of Z
- inverse value of Z
- same idea as sigmoid => with exponential decay

**Review by haugeSander — APPROVED** [review]
LGTM!

**Pull request merged**
